### ISSUE TITLE
Missing tf.* methods

### ISSUE THREAD

**Issue by Nucs** [SciSharp/TensorFlow.NET#356]
After reviewing `tf.*` methods, It seems that we miss a lot of generated functions that are available in other classes like `gen_ops`, `ops`, `variable_scope`, `control_flow_ops`.
From my understanding, the classes mentioned above are generated.
But during generation, the `@tf_export` attribute was ignored. 
`@tf_export` provides us with information about where this method should be available, for example:
![image](https://user-images.githubusercontent.com/649919/63250504-7a939d00-c274-11e9-9a95-7ce0ea71253b.png)
`in_top_k` should be available as `tf.math.in_top_k` and `tf.nn.in_top_k`.

In summary, there are `>50%` missing methods that can be fairly easily generated.
Copy-pasting code from python almost always ends up broken because of the amount of missing APIs.

**Comment by BrendanMulcahy** [comment]
I just ran into this issue today again.  Trying to use tf.losses.* and most things are missing :(

**Comment by Nucs** [comment]
@BrendanMulcahy, Most of the time they are implemented, just not available under the expected namespace (usually `gen_ops` or `ops`), use github's search.
If it is still missing, open up and issue and we'll make sure to add it

**Comment by SebastienES** [comment]
can't find 
tf.tensor_scatter_*
tf. scatter_nd
